src/commands.cc

```diff
@@ -460,11 +460,24 @@ const CommandDesc force_edit_cmd = {
     edit<true>
 };
 
-const ParameterDesc write_params{
+template<bool accepts_force>
+const ParameterDesc write_params;
+
+template<>
+const ParameterDesc write_params<true>{
+    {
+        { "sync", { false, "force the synchronization of the file onto the filesystem" } },
+        { "method", { true, "explicit writemethod (replace|overwrite)" } },
+        { "force", { false, "Allow overwriting existing file with explicit filename" } }
+    },
+    ParameterDesc::Flags::SwitchesOnlyAtStart, 0, 1
+};
+
+template<>
+const ParameterDesc write_params<false>{
```

Comment:
I dont see much value in making those specializations of a common template, they can just be two separate globals, for example write_params and write_no_force_params. (Not fully satisfied with those names, but I think this is still better than introducing a template global and a boolean parameter).

Reply:
yeah that was a remnant of me trying to share most parameter descriptions, fixed